Makefile

```diff
@@ -0,0 +1,82 @@
+.PHONY: clean-pyc clean-build docs clean
+
+help:
+	@echo "clean - remove all build, test, coverage and Python artifacts"
+	@echo "clean-build - remove build artifacts"
+	@echo "clean-pyc - remove Python file artifacts"
+	@echo "clean-test - remove test and coverage artifacts"
+	@echo "lint - check style with pep8"
+	@echo "test - run tests quickly with the default Python"
+	@echo "test-all - run tests on every Python version with tox DISABLED"
+	@echo "assets - build all JS/CSS assets"
+	@echo "coverage - check code coverage quickly with the default Python"
+	@echo "docs - generate Sphinx HTML documentation, including API docs"
+	@echo "release - package and upload a release"
+	@echo "dist - package locally"
+	@echo "install - install the package to the active Python's site-packages"
+
+clean: clean-build clean-pyc clean-test
+
+clean-build:
+	rm -fr build/
+	rm -fr dist/
+	rm -fr .eggs/
+	rm -fr dist-packages/
+	rm -fr dist-packages-temp/
+	find . -name '*.egg-info' -exec rm -fr {} +
+	find . -name '*.egg' -exec rm -f {} +
+
+clean-pyc:
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+	find . -name '__pycache__' -exec rm -fr {} +
+
+clean-test:
+	rm -fr .tox/
+	rm -f .coverage
+	rm -fr htmlcov/
+
+lint:
+	pep8 kalite
+
+test:
+	python setup.py test
+
+test-all:
+	@echo "Not supported yet"
+	# tox
+
+coverage:
+	coverage run --source kalite_gtk setup.py test
+	coverage report -m
+	coverage html
+	open htmlcov/index.html
+
+docs:
+	# rm -f docs/ka-lite.rst
+	# rm -f docs/modules.rst
+	# sphinx-apidoc -o docs/ ka-lite-gtk
+	$(MAKE) -C docs clean
+	$(MAKE) -C docs html
+	# open docs/_build/html/index.html
+
+assets: clean
+	# Necessary because NPM may have wrong versions in the cache
+	npm cache clean
+	npm install --production
+	node build.js
+	bin/kalite manage compileymltojson
```

Comment:
Won't work, since Windows can't associate `bin/kalite` with python and doesn't understand the `#!` notation.